e2e-tests/gef/cypress/e2e/application-activities-AIN.spec.js

```diff
@@ -17,7 +17,8 @@ let dealId;
 
 context('Submit AIN deal and check portalActivities', () => {
   before(() => {
-    cy.reinsertMocks();
+    cy.loadData();
+    cy.loadData();
```

Comment:
```suggestion
    cy.loadData();
```
I don't think we meant to have this twice